### PR TITLE
logrotate

### PR DESCRIPTION
**Pull request by jiayuehua**
#### What type of PR is this?
- [ ] bug
- [*] feature
- [ ] enhancement

#### What does this PR do?
logrotate rotate log 

#### Which issue(s)/PR(s) this PR relates to?
#3066 
  
#### Special notes for your reviewer, ex. impact of this fix, etc:
do not merge util thirdparty 3.0 is merged.

#### Checklist:
- [x] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

File: src/daemons/SetupLogging.cpp [hunk 1]
```diff
@@ -16,12 +18,17 @@ DEFINE_string(stderr_log_file, "stderr.log", "Destination filename of stderr");
...
 using nebula::fs::FileUtils;
 
-Status setupLogging() {
+Status setupLogging(const std::string & exe) {
   // If the log directory does not exist, try to create
-  if (!FileUtils::exist(FLAGS_log_dir)) {
-    if (!FileUtils::makeDir(FLAGS_log_dir)) {
-      return Status::Error("Failed to create log directory `%s'", FLAGS_log_dir.c_str());
-    }
+  if (!FileUtils::exist(FLAGS_log_dir) and !FileUtils::makeDir(FLAGS_log_dir)) {
```
Comment (yixinglu) [inline src/daemons/SetupLogging.cpp:23]: and => &&

File: src/daemons/SetupLogging.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
```
Comment (yixinglu) [inline src/daemons/SetupLogging.h:1]: 2021

**Review by yixinglu — APPROVED** [review]

File: conf/nebula-metad.conf.default [hunk 3]
```diff
@@ -20,6 +20,8 @@
 --stderr_log_file=metad-stderr.log
 # Copy log messages at or above this level to stderr in addition to logfiles. The numbers of severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively.
 --stderrthreshold=2
+# wether logging files' name contain time stamp, If Using logrotate to rotate logging files, than should set it to true.
```
Comment (critical27) [inline conf/nebula-metad.conf.default:23]: typo whether

**Review by critical27 — COMMENTED** [review]
Generally LGTM, a typo.

**Review by critical27 — APPROVED** [review]

**Pull request merged**